minos/common/database/database_pool.py

```diff
@@ -0,0 +1,26 @@
+"""
+Copyright (C) 2021 Clariteia SL
+
+This file is part of minos framework.
+
+Minos framework can not be copied and/or distributed without the express permission of Clariteia SL.
+"""
+
+import aiomisc
+import aiopg
+
+
+class AiopgPool(aiomisc.PoolBase):
+    def __init__(self, dsn, maxsize=10, recycle=60):
+        super().__init__(maxsize=maxsize, recycle=recycle)
+        self.dsn = dsn
+
+    async def _create_instance(self):
+        return await aiopg.create_pool(dsn=self.dsn)
+
+    async def _destroy_instance(self, instance: aiopg.Pool):
+        instance.close()
+        await instance.wait_closed()
+
+    async def _check_instance(self, instance: aiopg.Pool):
+        return instance.closed()
```

Comment:
Hi @vladyslav-fenchak, this implementation is not fine because what you should create and destroy is not the `Pool`, but the `connection` instead.

I mean, you can keep a `pool` instance as an instance attribute (in a similar way as the pool was defined at `PosgreSQLDataBase` class previously and what you should perform in the `*_instance` methods is the creation, destruction and checking of the connections according to the `aiopg.Pool` class. 

Documentation: https://aiopg.readthedocs.io/en/stable/core.html#aiopg.Pool